### DISCUSSION TITLE
Python modules

### DISCUSSION THREAD

**Discussion by Arijit-crypt** [atom/atom#22712] [Q&A]
I have just installed Atom and installed the script package also. I am trying to import the ```requests``` and the ```JSON`` module, but it shows an error every time while importing it.What should I do to resolve this? I am a Windows 10 user.

**Comment by icecream17** [comment]
Did you try `pip3 install requests` and `pip3 install json` or something like that?

(Btw, for the future, when there's an error it's usually helpful to copy/paste that error.)
